third_party/material/app.scss

```diff
@@ -9,3 +9,5 @@
 @use '@material/ripple/mdc-ripple';
 @use '@material/select/mdc-select';
 @use '@material/textfield/mdc-text-field';
+@use "@material/fab";
```

Comment:
Let's keep the alphabetical order, and also the detailed pattern here (`@material/fab/mdc-fab`?).

Reply:
SG.